doc/accessibility/index.md

```diff
@@ -0,0 +1,118 @@
+# Accessibility
+
+!!! note 
+Work in progress, request for discussion
+
+
+**Status**: first sketch, work in progress, request for collaboration
+
+**Date**: Updated 2025-04-03
+
+**Governance**: To Be Discovered; potentially a combo of this repo participants, DHCW UI/UX, NHS Wales UCB peers, etc.
+
+## Context
+
+We want to ensure accessibility for users, including users with disabilities. This is a fundamental aspect of providing an inclusive user experience, and an ethical principle, and a legal requirement.
```

Comment:
e.g. re-word: "Our goal is to create an accessible experience that includes and supports users with diverse abilities." noting that "when we build this way, evidence suggests it improves the experience for everyone."

Reply:
I'll ask disability wales. (My experience with Wales & disability & tech is that more actual people with disabilities recommend the direct use of the words "with disability" and similar, rather than "diverse abilities" or "varied abilities" etc.)